ReactNativeTraining/src/app/screens/components/ContactList/index.js

```diff
@@ -7,12 +7,15 @@ import styles from './styles';
 
 export default class ContactList extends Component {
   separatorComponent = () => <View style={styles.greyLine} />;
+
+  handlePress = item => this.props.onClick(item);
+
   renderItem = item => (
     <ListItem
       hideChevron
       roundAvatar
       avatar={item.avatar}
-      onPress={() => this.props.onClick(item)}
+      onPress={() => this.handlePress(item)}
```

Comment:
`onPress={this.handlePress}`. U dont need to declare another arrow function.

Reply:
But how 'handlePress' knows which item was clicked?

Reply:
Sry, onPress={this.handlePress(item)} and above`handlePress = item => {...}`